NewFile.kt

```diff
@@ -0,0 +1 @@
+lol
```

Comment:
<!--
  1 failure:  File contains no ...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: NewFile.kt;
  Line: 1;
-->  
- :no_entry_sign: File contains no license or license is improperly formatted.